compiler/passes/checkParsed.cpp

```diff
@@ -314,6 +317,21 @@ checkFunction(FnSymbol* fn) {
   }
 }
 
+static void nestedName(ModuleSymbol* mod) {
+  if (mod->defPoint == NULL) {
+    return;
+  }
+
+  ModuleSymbol* parent = mod->defPoint->getModule();
+  if (parent != mod &&
```

Comment:
parent == mod seems pretty weird. Does that have something to do with internal/standard module ?